### ISSUE TITLE
ionic native ios cordova-plugin-networkinterface not working.

### ISSUE THREAD

**Issue by ohtanzcom** [danielsogl/awesome-cordova-plugins#4452]
**I'm submitting a ...**  (check one with "x")
[x ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use the discussions section https://github.com/danielsogl/awesome-cordova-plugins/discussions

**Current behavior:**
Works fine on android, but on IOS
"Native: tried calling NetworkInterface.getWiFiIPAddress, but the NetworkInterface plugin is not installed."
error and it does not work.

**Expected behavior:**

**Steps to reproduce:**
```
ionic cordova plugin add cordova-plugin-networkinterface
npm install @awesome-cordova-plugins/network-interface
```

**Other information:**

Reinstallation does not improve the situation.

```
ionic cordova platform rm ios
ionic cordova plugin remove cordova-plugin-networkinterface
npm uninstall @awesome-cordova-plugins/network-interface
ionic cordova plugin add cordova-plugin-networkinterface
npm install @awesome-cordova-plugins/network-interface
ionic cordova platform add ios
```

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Ionic:

   Ionic CLI                     : 6.20.4 (/Volumes/DATA/Users/owner/.nodebrew/node/v16.13.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 6.3.9
   @angular-devkit/build-angular : 15.0.2
   @angular-devkit/schematics    : 15.0.2
   @angular/cli                  : 15.0.2
   @ionic/angular-toolkit        : 6.1.0

Cordova:

   Cordova CLI       : 11.0.0
   Cordova Platforms : ios 6.2.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 5.0.0, (and 27 other plugins)

Utility:

   cordova-res : 0.15.4
   native-run  : 1.7.1

System:

   Android SDK Tools : 26.1.1 (/Volumes/DATA/Users/owner/Library/Android/sdk)
   ios-deploy        : 1.11.4
   ios-sim           : 8.0.2
   NodeJS            : v16.13.0 (/Volumes/DATA/Users/owner/.nodebrew/node/v16.13.0/bin/node)
   npm               : 8.1.0
   OS                : macOS Monterey
   Xcode             : Xcode 14.1 Build version 14B47b
```

**Comment by ohtanzcom** [comment]
I re-created it from the project folder and it was fixed...🤔

**Comment by arbassic** [comment]
> I re-created it from the project folder and it was fixed...🤔

What do you mean by re-created it from the project folder?

I have similar problem with `InAppPurchase2 plugin is not installed.`, but for a Capacitor-powered project...

**Comment by ohtanzcom** [comment]
@arbassic 
I thought maybe a configuration file somewhere was broken...?
So I re-created the project itself.

```
mv ProjectName ProjectName.bak
ionic start ProjectName blank --type=angular
cd ProjectName
ionic cordova plugin add cordova-plugin-networkinterface
npm install @awesome-cordova-plugins/network-interface
...(other plugins)
ionic cordova platform add ios
```

**Comment by grantmk** [comment]
> > I re-created it from the project folder and it was fixed...🤔
> 
> What do you mean by re-created it from the project folder?
> 
> I have similar problem with `InAppPurchase2 plugin is not installed.`, but for a Capacitor-powered project...

Have exactly the same problem with the same plugin ..... completely removed Xcode and reinstalled b ut didn't help at all!

**Comment by grantmk** [comment]
10 minutes after posting I found the solution. 

Check your "cordova-plugin-purchase" package version (it's probably v13.x.x). I rolled back to v11.0.0 and now everything works! 11 is actually one major version before 13 so is not that old and supports Google billing v4 so all good!

**Comment by arbassic** [comment]
Thanks for sharing the workaround!

However, meanwhile I found another plugin, made directly for capacitor from [CapGo](https://github.com/Cap-go/capacitor-purchases). It uses RevenueCat and the second Cordova plugin under the hood, but after setup (which took some time) everything is working well. And the interface is a lot more modern, cleaner and a way simpler (i.e. awaits). One thing I found is that "setup" never resolves on Android, but if you just go on, everything works well. And yes, it works with standard consumables / single use items (not only subscriptions)

**Comment by grantmk** [comment]
Interesting, thanks for that find. I actually started with Cordova purchases, then tried RevenueCat (their SDKs directly). My use case is a subscription that awards X credits per month / year - managed by my own backend DB.

After the pain of implementation I discovered that RevCat doesn't provide the unique transaction identifier "web_order_line_item_id" that you get when going native with Cordova and Apple IAP webhooks. In fact I found it extremely difficult to marry up the renewal webhook on RevCat and the customers subscription.

So I'm coming back to native Cordova implementation. I still cannot believe that RevCat don't include the Apple transaction receipt in their webhook!
